packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -105,11 +114,15 @@ class WebEntrypointTarget extends Target {
 
 /// Compiles a web entry point with dart2js.
 abstract class Dart2WebTarget extends Target {
-  const Dart2WebTarget(this.webRenderer);
+  const Dart2WebTarget();
 
-  final WebRendererMode webRenderer;
   Source get compilerSnapshot;
 
+  WebCompilerConfig get compilerConfig;
+
+  Map<String, dynamic> get buildConfig;
```

Comment:
Per [the style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-var-and-dynamic), consider using `Object` or `Object?` in place of `dynamic`.